image/roxctl/Dockerfile.rhtap

```diff
@@ -0,0 +1,53 @@
+# TODO: can we follow tags?
+FROM registry.access.redhat.com/ubi8/go-toolset:1.19 as builder
```

Comment:
Could you please try the following?

```
FROM brew.registry.redhat.io/rh-osbs/openshift-golang-builder:rhel_8_1.20 as builder
```

`brew.registry.redhat.io` is available from the outside world so it's a matter of pull secrets, if RHTAP has compatible ones.

FWIW, I asked a question in Slack about availability of the new Go. https://redhat-internal.slack.com/archives/C04PZ7H0VA8/p1696606334046349

Reply:
Indeed, that's the way https://redhat-internal.slack.com/archives/C04L50S5XM4/p1696606623923649

If `brew.registry.redhat.io/` does not work for some reason, there is `registry-proxy.engineering.redhat.com/` alias, but I think `brew.*` should work.

Reply:
Per the linked issue https://issues.redhat.com/browse/RHELPLAN-167618:

> Our current guidance is therefore to direct internal users to openshift-golang-builder, and external users to CentOS Stream to access pre-release builds of Go.

Reply:
`brwe.registry.redhat.io` worked after adding a new service account per [this](https://access.redhat.com/RegistryAuthentication#creating-registry-service-accounts-6) and adding the username+password as Build secrets in RHTAP. 

Changes in https://github.com/stackrox/stackrox/pull/7982/commits/8a44b311bffbd1761730eccb460cc776cce5a574. 

What does not work correctly is the `deprecated-base-image-check`. The brew image cannot be investigated through Pyxis, it still shows this step as passed.

Reply:
IMHO the comments in this file sufficiently summarize this discussion and it can be resolved.